src/targets/newbeedrone_2400.ini

```diff
@@ -0,0 +1,14 @@
+# ********************************
+# Receiver NewBeeDrone_2400_RX
+# ********************************
+
+# ELRS-RX24D
+[env:NewBeeDrone_D_2400_RX_via_UART]
+extends = env:Unified_ESP8285_2400_RX_via_UART
+board_config = newbeedrone.rx_2400.rx24d
+
+[env:NewBeeDrone_D_2400_RX_via_BetaflightPassthrough]
+extends = env:NewBeeDrone_2400_RX_R24D_via_UART
+
+[env:NewBeeDrone_D_2400_RX_via_WIFI]
+extends = env:NewBeeDrone_2400_RX_R24D_via_UART
```

Comment:
Not required any more as you can flash a Unified target from platformio and select the layout from the menu that is displayed at the end of the build.